analysis/deep2_r23_o32/zoom_and_gauss_general.py

```diff
@@ -444,6 +443,12 @@ def zm_general(dataset, fitspath, stack2d, wave, lineflag, dispersion,
 
     s2 = 5.0  # a fitting requirement
 
+    # Import Stacking Dictionaries
+    fits_dict = general.read_fitsfiles(outfile_grid)
+    stack2d = fits_dict['fits_data']
+    wave = fits_dict['wave']
+    dispersion = fits_dict['dispersion']
```

Comment:
As described in #72, I recommend passing `fits_dict` into `zoom_gauss_plot` rather than splitting it out and providing `stack2d`, `wave`, and `dispersion`.

Reply:
fixed